### PR TITLE
docs: update laravel sanctum docs URL

### PR DESCRIPTION
**Pull request by ayoubbajjou**
Update `Laravel sanctum` docs URL to redirect to the latest version of `Laravel` currently (`9.x`), so we don't need to specify the version on every upgrade of the Framework anymore this will allow us to always redirect the user to the latest version.

### CODE REVIEW

**Comment by ayoubbajjou** [comment]
@Intevel could you check this simple update, please!

**Review by Intevel — APPROVED** [review]
v9.x is the newer version, you can also change the version on the Laravel page. **Approved! Looks simple**

**Comment by ayoubbajjou** [comment]
@Intevel Thanks for approving this first PR in contributing to open source even if it's simple like this, So nothing to do from my side in this PR, right?

**Comment by Intevel** [comment]
No, we have to wait until a maintainer merges it. But currently looks bad because no one works _active_ on this module

**Comment by ayoubbajjou** [comment]
Ok, no problem let's wait, Thanks again

**Comment by bmulholland** [comment]
Thanks for the fix :)

**Pull request merged**
